src/chigame/users/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
-
+from .views import UserListView
 from chigame.users.views import user_detail_view, user_redirect_view, user_update_view
 
 app_name = "users"
 urlpatterns = [
     path("~redirect/", view=user_redirect_view, name="redirect"),
     path("~update/", view=user_update_view, name="update"),
     path("<int:pk>/", view=user_detail_view, name="detail"),
+    path('users/', UserListView.as_view(), name='user-list'),
```

Comment:
Why are the API URL's not located in the `chigame/src/api` app? 
Since the resulting URL for this endpoint is `chigame/users/users/` which is more confusing than `chigame/api/users`.
Since there are other URLs that are/will be associated with the users app (such as profile, friends, etc...), so adding API endpoints in the mix will then confuse the possible paths are associated with users.

Please refactor your code to put locate your endpoint at `api/users` instead of `users/users`.

Reply:
The API URL's have been moved accordingly